cmd/ausoceantv/main.go

```diff
@@ -105,9 +106,19 @@ func main() {
 	// Create app.
 	app := fiber.New(fiber.Config{ErrorHandler: api.ErrorHandler})
 
-	// Perform one-time setup or bail.
+	// Encrypt cookies.
+	// NOTE: This must be done before any middleware which uses cookies.
 	ctx := context.Background()
-	svc.setup(ctx, app)
+	key, err := gauth.GetSecret(ctx, projectID, "sessionKey")
+	if err != nil {
+		log.Fatalf("unable to get sessionKey secret: %v", err)
```

Comment:
This seems more like a programmer error (?) in which case we should use panic (?). We can also recover panics (plus do fancy things like email notify), and it show a stacktrace. ```log.Fatalf``` on the other hand uses ```os.Exit(1)``` which is pretty harsh.